modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -172,6 +172,22 @@ public ObjectField deleteObjectField(ObjectField objectField)
 		return _deleteObjectField(objectField);
 	}
 
+	@Override
+	public void deleteObjectFieldByObjectDefinitionId(Long objectDefinitionId)
+		throws PortalException {
+
+		for (ObjectField objectField :
+				objectFieldLocalService.getObjectFields(objectDefinitionId)) {
+
+			if (Validator.isNull(objectField.getRelationshipType())) {
+				objectFieldPersistence.remove(objectField);
```

Comment:
Could we call the method deleteObjectFIeld from this same class instead of calling the persistence?
I'm trying to avoid calling directly the persistence class.

Reply:
If I call the LocalService delete, it do verifications that throw exceptions in a lot Integration tests scenarios